leshan-server-demo/src/main/java/org/eclipse/leshan/server/demo/servlet/ClientServlet.java

```diff
@@ -194,7 +193,7 @@ protected void doPut(HttpServletRequest req, HttpServletResponse resp) throws Se
             LOG.warn("Invalid request or response", e);
             resp.setStatus(HttpServletResponse.SC_BAD_REQUEST);
             resp.getWriter().append(e.getMessage()).flush();
-        } catch (ResourceAccessException | RequestFailedException e) {
+        } catch (ResourceAccessException e) {
```

Comment:
We should replace RequestFailedException by RequestRejectedException else we change the behavior and I'm not sure this is what we want.